src/index.html

```diff
@@ -9,9 +9,216 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
   </head>
+
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      target="_blank"
+    >
+      <fieldset class="personalInfo form-set">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+
+          <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+
+          <input
+            type="text"
+            id="name"
+            name="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+
+          <input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            placeholder="12"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="birth">Full date of birth:</label>
+
+          <input
+            type="date"
+            id="birth"
+            name="date"
+            min="1923-01-01"
+            max="2020-12-31"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="terms">I accept the term of the agreement</label>
+
+          <input
+            type="checkbox"
+            id="terms"
+            name="terms"
+            required
+            checked
+          >
+        </div>
+      </fieldset>
+
+
+      <fieldset class="registaration form-set">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+
+          <input
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password:</label>
+
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="8"
+            maxlength="16"
+            pattern="\d+"
+            title="Please, use only numbers."
+            required
+          >
+        </div>
+      </fieldset>
+
+
+      <fieldset class="interestingFacts form-set">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label class="cat">
```

Comment:
Why do you need this class?
```suggestion
          <label>
```